### DISCUSSION TITLE
can't input due date on board mode ( beta project )

### DISCUSSION THREAD

**Discussion by jeonhoeun** [community/community#8373] [Projects and Issues]
I'm using project(beta).
And I use both view mode ( table, board)

When the view mode is table, I can input due date on draft.
But I can't input dut date on draft when the view mode is board.

I want input due date on draft during the view mode is board.

(switching view mode continuously is not funny work... TT)

**Comment by maxcomperatore** [comment]
Two key technical hurdles explain why this is behaving like that between Reaper, Dexed, and sysex:

1. The VST3 specification deliberately dropped arbitrary SysEx streaming
Steinberg explicitly removed raw MIDI 1.0 byte streaming (0xF0 ... 0xF7) from the base VST3 spec. VST3 replaced it with normalized parameter changes. While some DAWs (like Reaper) can package sysex into VST3 event queues, most VST3 plugins simply drop sysex on track playback unless specifically coded with a custom IDataEvent queue. If you switch Reaper to load the VST2 build of Dexed instead, the raw MIDI stream bypasses that constraint and passes sysex directly to the plugin instance.

2. VCED vs VMEM format differences
A single voice edit buffer dump (VCED) is 155 bytes unpacked (one byte per parameter, envelopes, algorithm, feedback, LFO, and the 10-char ASCII name). 
A 32-voice bank (VMEM) is 4,096 bytes of packed voice data (32 voices x 128 bytes each) plus the 6-byte Yamaha header `F0 43 00 09 20 00`, 1-byte checksum, and `F7` (total 4,104 bytes).
In VMEM, pairs of parameters (like coarse frequency and osc mode, or detune and fine frequency) are bit-packed into single bytes. If Dexed receives an unpacked 155-byte VCED stream when expecting a packed 4,104-byte bank, it discards it without an error message.

We dealt with this exact conversion headache when building the Web MIDI engine for [bipluk](https://bipluk.com). If you ever need to inspect whether a DX7 sysex bank has valid checksums, properly packed VMEM blocks, or clean ASCII names, you can drop the .syx file right into the browser to verify the structure.

**Comment by tylerharris82836-netizen** [comment]
Having the due date available directly on draft cards would make the board workflow much smoother, especially for teams handling [cold calling services india ](https://pearllemonleads.in/services/cold-calling/)without constantly switching views.

**Comment by ArulOly** [comment]
I agree with this. Having to switch from Board view to Table view just to set a due date on a draft is inconvenient.

It would be much more useful if draft cards in Board view supported the same due-date editing available in Table view. A date field or quick-edit option directly on the card would make the workflow much smoother.

Hopefully GitHub can consider adding this to Projects.
